liquibase-core/src/main/java/liquibase/ui/interactive/InteractivelyPromptableParameters.java

```diff
@@ -0,0 +1,83 @@
+package liquibase.ui.interactive;
+
+import liquibase.ui.interactive.getter.DirectoryGetter;
+import liquibase.ui.interactive.getter.EnumGetter;
+import liquibase.ui.interactive.getter.FilenameGetter;
+import liquibase.ui.interactive.getter.StringGetter;
+
+import java.util.Arrays;
+
+import static liquibase.integration.commandline.LiquibaseCommandLineConfiguration.DEFAULTS_FILE;
+
+/**
+ *
+ */
+public enum InteractivelyPromptableParameters implements IInteractivelyPromptableEnum {
+    PROJECT_DIR(null, "Enter a relative path to desired project directory, or (n) to cancel:", ".", (String)null, new DirectoryGetter()),
```

Comment:
This all seems to be "init" specific logic and prompts, not something that belongs in the global liquibase.ui.interactive API.

Reply:
Not sure I agree here given the future vision for this system. When this interactive flow is enabled for `liquibase update` (and other core commands), these parameters will be called from multiple locations.